pdblp/pdblp.py

```diff
@@ -731,6 +745,44 @@ def bsrch(self, domain):
                     data.append(f.getElementAsString("StringValue"))
         return pd.DataFrame(data)
 
+    def secf(self, query, max_results=10, yk_filter="NONE"):
+        """
+        This function uses the Bloomberg API to retrieve Bloomberg
+        SECF Data queries. Returns list of tickers.
+
+        Parameters
+        ----------
+        query: string
+            A character string representing the desired query. Example "IBM"
+        max_results: int
+            Maximum number of results to return. Default 10. 
+        yk_filter: string
+            A character string respresenting a Bloomberg yellow-key to limit
+            search results to. Valid values are: CMDT, EQTY, MUNI, 
+            PRFD, CLNT, MMKT, GOVT, CORP, INDX, CURR, MTGE. Default NONE.
+
+        Returns
+        -------
+        data: list
+            List of bloomberg tickers from the SECF function
+        """
+        logger = _get_logger(self.debug)
+        request = self.instrService.createRequest("instrumentListRequest")
+        request.set("query", query)
+        request.set("maxResults", max_results)
+        request.set("yellowKeyFilter", "YK_FILTER_%s" % yk_filter)
+        logger.info("Sending Request:\n%s" % request)
+        self._session.sendRequest(request, identity=self._identity)
+        data = []
+        for msg in self._receive_events():
+            for r in msg.getElement("results").values():
+                ticker = r.getElementAsString("security")
+                ticker = ticker.replace('<', ' ').replace('>', '').upper()
```

Comment:
I think it is better to leave this line out. The general philosophy of `pdblp` is to provide a thin wrapper around `blpapi` and limit the data processing. If a user wants this they can do it easily enough afterwords using `df.ticker.str.replace("<", "").str.replace(">", "").str.upper()`